### PR TITLE
[ScaleType] add parallax scale type

### PR DESCRIPTION
**Pull request by woyunowuyuda**
## Motivation (required)
The material design image type. Refers to https://material.io/design/material-studies/basil.html.
Make it easy to have a parallax image animation.

## Test Plan (required)

set the scale-type to PARALLAX.

## Demo
hor

ver

### CODE REVIEW

File: drawee/src/main/java/com/facebook/drawee/view/SimpleDraweeView.java [hunk 1]
```diff
@@ -208,4 +208,9 @@ public void setActualImageResource(@DrawableRes int resourceId, @Nullable Object
   public void setImageResource(int resId) {
     super.setImageResource(resId);
   }
+
+  @Override
+  public boolean performClick() {
+    return super.performClick();
+  }
```
Comment (erikandre) [inline drawee/src/main/java/com/facebook/drawee/view/SimpleDraweeView.java]: Please remove this

**Comment by erikandre** [comment]
Thanks for the pull request @woyunowuyuda !

**Comment by woyunowuyuda** [comment]
@erikandre I have deleted it

**Comment by oprisnik** [comment]
I'm going to close this pull request for now. Please feel free to submit a new one on the basis of my previous comment.

**Pull request closed without merging**